Assets/Scripts/KeyLocationController.cs

```diff
@@ -6,55 +6,74 @@
 
 public class KeyLocationController : MonoBehaviour
 {
-    private float radius;
+    public float radius;
     public GameObject sphere;
+    public GameObject clearSphere;
+    public int clearSphereSpawnAmount;
     public List<GameObject> players = new List<GameObject>();
-    private float speedMod;
+    public float speedMod;
+    public int health = 10;
+    private int shrinkValue = 20;
+    private bool isDestroyed = false;
 
     void Start()
     {
         StartCoroutine(GetPlayers());
-        radius = (sphere.transform.localScale.x/2)-1; //-1 to reduce screen clipping with sphere
+        sphere.transform.localScale = new Vector3((radius * 2) + 1, (radius * 2) + 1, (radius*2) + 1); //+1 to reduce screen clipping with sphere
```

Comment:
why can't you just have radius be set a bit bigger in general and not multiply by 2 here? Just asking to understand the reasoning, I imagine there's something important about it 🤔

Reply:
The value used for the dist calc is half of the value used for the sphere size since scale essentially is diameter. Either we *2 the sphere, or we /2 the value in dist calculation.

Reply:
This makes sense 👍